### DISCUSSION TITLE
What does Update do exactly ? Does bird recognition get updated?

### DISCUSSION THREAD

**Discussion by mattgolde** [Nachtzuster/BirdNET-Pi#263] [General]
Hi,

I was wondering if birdnet-pi ever updates its bird recognizing data ?
I see there is an update option, "Tools" > "System Controls" > "Update", this appears to sync the install with this (Nachtzuster) git repo - but it is unclear if this updates any bird data to improve recognition.

And further to that, if it does, where does that come from ? eBird ?

TIA
Matt
(PS we are having lots of fun watching bird data - 2 stations so far going in Auckland NZ)

**Comment by tphakala** [comment]
BirdNET Analyzer is upstream project for AI recognition model used in BirdNET-Pi https://github.com/kahst/BirdNET-Analyzer, they have not released updated model since June 2023

**Comment by mattgolde** [comment]
I see there is a Jan 2024 model available too, which birdnet-pi seems to make available.

I see the analyzer allows you to add/train it for more bird calls, so I guess you could drop your own custom model onto an instance of birdnet-pi ?

**Reply by tphakala** [reply]
That Jan 2024 model is a "meta model" or "range filter model". It contains probabilities of each species' occurrence based on geographic location and date. It has no impact on how well the BirdNET model recognizes species from audio.

It is indeed possible to train your own classifier (or model) with BirdNET Analyzer, in fact I am doing just that at the moment for my BirdNET-Go application. This, however is very slow and tedious task, and I question myself why I do this ..

**Reply by mattgolde** [reply]
ok thanks.

Yes I am not wanting to go down the training path unless I really have to - this is really just a hobby at the moment.
